### PR TITLE
[timepicker] CSS component

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Add desktop component covering default and outlined style and disabled state.
Incrementing buttons still need to be plugged by apps while the Angular component is still in development.

-----

https://user-images.githubusercontent.com/25581936/203570993-5e9bc8e8-51f1-4312-a085-a1da1de80e25.mov

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

File: stories/documentation/forms/timepicker/timepicker-basic.stories.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	},
+} as Meta;
+
+function getTemplate(args: timepickerBasicStory): string {
+		const classes = [args.mod, args.palette].filter(Boolean).join(' ');
+		const disabled = args.disabled ? `disabled` : '';
+	return `
+<fieldset class="timepicker ${classes}" ${disabled}>
+	<legend class="u-mask">Durée</legend>
+	<div class="timepicker-field">
```
Comment (vvalentin-lucca) [inline stories/documentation/forms/timepicker/timepicker-basic.stories.ts:38]: No implicit label around your input and its name?

File: stories/documentation/forms/timepicker/timepicker-basic.stories.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+		<span class="u-mask">heures</span>
+		<input class="timepicker-field-input" type="number" min="0" max="23" step="1" value="1" tabindex="1">
+		<button class="timepicker-field-increment" type="button" tabindex="-1" aria-hidden="true">
```
Comment (vvalentin-lucca) [inline stories/documentation/forms/timepicker/timepicker-basic.stories.ts:41]: Be careful if you put an implicit label, you will have to replace the buttons by links for the code to remain valid.

**Comment by vvalentin-lucca** [comment]
Capture d’écran 2022-11-30 à 10 58 24

With Firefox the input helpers hide the values.

**Comment by vvalentin-lucca** [comment]
Capture d’écran 2022-11-30 à 11 00 53

With iOS the buttons do not work, and the numbers are not fully visible.

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1913/storybook

**Comment by jeremie-lucca** [comment]
Fixed ui issues (even on iOS after downloading 20+ gb 🙄)
@vvalentin-lucca Buttons will be plugged with the future NG component.

File: stories/documentation/forms/timepicker/timepicker-basic.stories.ts [hunk 3]
```diff
@@ -36,8 +36,10 @@ function getTemplate(args: timepickerBasicStory): string {
 <fieldset class="timepicker ${classes}" ${disabled}>
 	<legend class="u-mask">Durée</legend>
 	<div class="timepicker-field">
-		<span class="u-mask">heures</span>
-		<input class="timepicker-field-input" type="number" min="0" max="23" step="1" value="1" tabindex="1">
+		<label>
+			<span class="u-mask">heures</span>
```
Comment (vvalentin-lucca) [inline stories/documentation/forms/timepicker/timepicker-basic.stories.ts:40]: No capitalization?

**Pull request merged**